app/subscriber/src/features/search-page/components/advanced-search/AdvancedSearch.tsx

```diff
@@ -129,14 +122,22 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({ onSearch }) =>
   const handleSearch = async () => {
     const updatedFilterSettings = {
       ...search,
-      search: controlled,
+      search: searchTerms,
     };
 
     storeSearchFilter(updatedFilterSettings);
 
     onSearch?.(updatedFilterSettings);
   };
 
+  React.useEffect(() => {
```

Comment:
You need to use these to update state on load because of delays in ajax requests